CHANGELOG.md

```diff
@@ -22,6 +22,8 @@
   Add `Validation` data type to Extra modules.
 * [#103](https://github.com/kowainik/relude/pull/104):
   Drop utf8-string dependency.
+* [#109](https://github.com/kowainik/relude/issues/109):
+  Use Dhall v3.0.0 for hlint file generation.
```

Comment:
I think it's a good idea to update `README.md` as well to tell that at least Dhall 3.0.0 is required.

Also, I think it's okay to add this to CHANGELOG. I think that changelog can have multiple roles: inform users about interface changes and also keep track of things we did. It's always good to see what has been done :slightly_smiling_face: